core/templates/dev/head/pages/Base.js

```diff
@@ -28,6 +29,12 @@ oppia.controller('Base', [
     $scope.isSidebarShown = SidebarStatusService.isSidebarShown;
     $scope.closeSidebarOnSwipe = SidebarStatusService.closeSidebar;
 
+    // Listner to catch the change in language prefrence
+    $rootScope.$on('$translateChangeSuccess', function(evt, translationResponse) {
+      var CURRENT_LANG = translationResponse.language;
```

Comment:
This isn't a constant. Why not just do

$scope.currentLang = translationResponse.language?